packages/pipeline-editor/src/PipelineService.tsx

```diff
@@ -118,7 +118,7 @@ export class PipelineService {
     return RequestHandler.makePostRequest(
       'elyra/pipeline/schedule',
       JSON.stringify(pipeline),
-      this.getWaitDialog()
+      this.getWaitDialog('Preparing and running pipeline ...')
```

Comment:
Should this be more like `Packaging and submitting pipeline...`?

Reply:
```suggestion
      this.getWaitDialog('Packaging and submitting pipeline ...')
```